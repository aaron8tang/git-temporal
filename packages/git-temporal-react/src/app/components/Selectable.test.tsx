import React from 'react';
import { shallow } from 'enzyme';

import { Selectable } from './Selectable';

describe('components/Selectable', () => {
  describe('when rendered with required props', () => {
    let wrapper;
    const onClickFn = jest.fn();
    beforeAll(() => {
      wrapper = shallow(
        <Selectable onClick={onClickFn}>this is the a menu item</Selectable>
      );
    });

    test('it should match snapshot (it should have rendered a label and a text element)', () => {
      expect(wrapper).toMatchSnapshot();
    });
  });
});
