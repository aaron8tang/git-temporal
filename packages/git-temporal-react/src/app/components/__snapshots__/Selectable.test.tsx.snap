// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Selectable when rendered with required props it should match snapshot (it should have rendered a label and a text element) 1`] = `
<div
  onClick={[Function]}
  onMouseLeave={[Function]}
  onMouseOver={[Function]}
  style={
    Object {
      "border": "1px solid transparent",
      "color": "#333333",
      "fontSize": 12,
      "fontWeight": "normal",
      "lineHeight": 1.35,
      "padding": 5,
    }
  }
>
  this is the a menu item
</div>
`;
